src/shared/state/api-slices/restSlice.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export default createApi({
+  reducerPath: 'restApi',
+  baseQuery: fetchBaseQuery({ baseUrl: '/' }),
```

Comment:
You will notice that there's a warning in the terminal:

```
Warning: `fetch` is not available. Please supply a custom `fetchFn` property to use `fetchBaseQuery` on SSR environments.
```

Let's provide the `fetch` from the `cross-fetch` package that we have as a `fetchFn` option:

```
import fetch from 'cross-fetch'

...

baseQuery: fetchBaseQuery({ baseUrl: '/', fetchFn: fetch }),
```